src/text_engine/effects/GameCharacterEffect.java

```diff
@@ -19,10 +19,16 @@ public void accept(GameCharacter gameCharacter) {
         this.consumer.accept(gameCharacter);
     }
 
+    public static final GameCharacterEffect NULL = new GameCharacterEffect(
```

Comment:
Because GameEntityEffects can't exist. Every effect has to take in a GameCharacter and then find its relevant class off of that.